### PR TITLE
Adjusted GB-Appointed Seats Election Timeline.md

### PR DESCRIPTION
**Pull request by Naomi-Wash**
Adjusted the GB appointed seat timeline back to the original dates.

### CODE REVIEW

File: elections/tac-and-scir-election-process.md [hunk 1]
```diff
@@ -106,23 +106,23 @@ About 10 months after the previous cycle, a committee of Election Officials will
...
+| December 1 | The voting period ends at 11:59 pm PST
+| December 2 | The GB-appointed TAC representatives announced
 
  **New members seated: January 1**
 
 
 ## Voter Registration form
 
-The electorate can request a ballot by filling out this [Google form](https://forms.gle/7suYexAnPxndvX856).
+The electorate can request a ballot by filling out this [Google Form](https://forms.gle/2FofZgurk8GzZb7h7).
```
Comment (steiza) [inline elections/tac-and-scir-election-process.md:127]: Nice catch!

**Review by steiza — APPROVED** [review]
Looks good to me - thanks!

**Review by marcelamelara — APPROVED** [review]
LGTM, thank you @Naomi-Wash !

**Review by lehors — APPROVED** [review]
Better! Thanks.

**Review by sevansdell — APPROVED** [review]
this is good!

**Pull request merged**